zazu/repo/commands.py

```diff
@@ -51,10 +55,33 @@ def ci(ctx):
 
 
 @repo.command()
+@click.argument('repository_url')
+@click.option('--nohooks', is_flag=True)
+@click.option('--nosubmodules', is_flag=True)
 @click.pass_context
-def clone(ctx):
-    """Clone and initialize a repo"""
-    raise NotImplementedError
+def clone(ctx, repository_url, nohooks, nosubmodules):
+    """Clone and initialize a repo
+
+    Args:
+        repository_url(str):url of the repository to clone
```

Comment:
you can add this documentation to the click options so it is rendered for you automatically on --help